community-membership.md

```diff
@@ -1,77 +1,198 @@
-  create/delete labels and milestones); technically can lgtm any PR and cause it
-  to be merged by the submit queue, but expected to respect OWNERS files; expected to review PRs, fix bugs, maintain and
-  improve health and quality of the project, provide user support, mentor and guide approvers,
-  reviewers, and other contributors; must apply to `kubernetes-maintainers@googlegroups.com`, with a
-  [Champion](https://github.com/kubernetes/community/blob/master/incubator.md#faq) from the existing
-  kubernetes-maintainers members and a Sponsor from Project Approvers, with a summary
-  of contributions to the project, current project responsibilities, and links to merged and assigned PRs;
-  at least 3 of the maintainers must approve the application, with no objections; the application
-  expires after 2 weeks if not enough approvals are granted
-- **Project Approvers**: approver in [top-level OWNERS file in kubernetes repo](https://github.com/kubernetes/kubernetes/blob/master/OWNERS);
-  de-facto project decision makers; technically can 
-  approve virtually any PRs; can sponsor incubator repos; can sponsor maintainers;
-  maintainer in good standing for at least 1 year; strong technical vision;
-  committed to project's mission and culture; nomination/application process TBD
-- [**API Approver**](https://github.com/orgs/kubernetes/teams/api-approvers):
-  lead designers of the project, who are familiar with the 
-  design, requirements, mechanics, conventions, style, scope, gotchas, etc. 
-  of the API; most beta/GA API changes are vetted by the API approvers
-- [**API Reviewer**](https://github.com/orgs/kubernetes/teams/api-reviewers):
-  contributors familiar with design, requirements, mechanics, conventions, style,
-  scope, gotchas, etc. of the API; have written and/or reviewed Kubernetes APIs
\ No newline at end of file
+## Member
+
+Members can have issues and PRs assigned to them and participate in SIGs.
+Members are expected to remain active contributors to the community.
+
+**Defined by:** Kubernetes GitHub organization outside collaborator (read access)
+
+### Requirements
+
+- Enabled [two-factor authentication](https://help.github.com/articles/about-two-factor-authentication/) on their GitHub account
+- Have contributed to the project or community.  Contribution may include, but is not limited to:
+  - Authoring or reviewing PRs on GitHub
+  - Filing or commenting on issues on GitHub
+  - Contributing to SIG or community discussions (e.g. meetings, slack, email discussion forums, stackoverflow, meetups)
+- Sponsored by 2 members
+  - TODO: Define the exact process for sponsorship.
+- Have read the [developer guide](contributors/devel/README.md)
+- Subscribed to kubernetes-dev@googlegroups.com
+- Some idea of what they intend to work on
+
+TODO: Define a process for reviewing memberships.
+
+### Responsibilities and privileges
+
+- Tests are automatically run for PullRequests from org members
```

Comment:
This is currently true.  We check org membership of the author before allowing tests to be run against a PR.  See [here](https://github.com/kubernetes/test-infra/blob/master/prow/plugins/trigger/pr.go#L38) if you're curious about the implementation